flair/nn/model.py

```diff
@@ -233,6 +233,14 @@ def print_model_card(self):
                 "trained or was trained with Flair version < 0.9.1)"
             )
 
+    @property
+    def supports_smaller_training_vocab(self) -> bool:
+        # the smaller training vocab expects classification tasks, otherwise it won't work.
+        return False
+
+    def get_used_tokens(self, corpus: Corpus) -> typing.Iterable[List[str]]:
+        pass
```

Comment:
Similar to above, this could either be moved into Classifier or replaced with a function. I worry a bit about "bloating" the top level interfaces with two new functions that only one feature needs.

Reply:
I feel like since these are not abstract methods, but implementations, it is not important to implement them, and therefore not bloated in terms of needs to implement.